### PR TITLE
Testing components 2

### PR DESCRIPTION
**Pull request by chadrick-d-dev**
### What’s this PR do?
* Creates passing tests for About, FindARoommate, and Thumbnails.
* Creates skipped, non passing, in progress tests for Profile
* Formats function components from es5 to es6 arrow functions

### Where should the reviewer start?
* Review test files

### How should this be manually tested?
* Run ```npm test``` to see if tests pass

### Any background context you want to provide?
* As developers we need to run testing on our ui to ensure that it functions correctly

### Questions
* Anyone see what's wrong with the tests in Profile.test.js

### CODE REVIEW

File: src/About/About.test.js [hunk 1]
```diff
@@ -1,20 +1,37 @@
...
+import { createMemoryHistory } from "history";
 
 describe('About', () => {
-  //this testing file will mostly be for unit testing
-  it.skip('It should display a description of the app and the collaboration', () => {
-    // setup and execution 
+  it('It should display a description of the app and the collaboration', () => {
+    
     render(
+      <MemoryRouter>
```
Comment (cameronRomo) [inline src/About/About.test.js:13]: Nice use of memory router!

Comment (chadrick-d-dev) [reply src/About/About.test.js:13]: 💅

File: src/FindARoommate/FindARoommate.test.js [hunk 2]
```diff
@@ -2,19 +2,18 @@ import React from 'react';
...
+import { getResidentsBySemester } from '../apiCalls';
+import { springResident } from '../mockResidentData';
 jest.mock('../apiCalls.js');
 
 describe('FindARoommate', () => {
-  let mockedResidents;
+  
   beforeEach(() => {
-    mockedResidents = getAllResidents.mockResolvedValue(mockResidents)
+    getResidentsBySemester.mockResolvedValueOnce(springResident)
```
Comment (cameronRomo) [inline src/FindARoommate/FindARoommate.test.js:14]: Fantastic mock!

File: src/Thumbnails/Thumbnails.test.js [hunk 3]
```diff
@@ -2,19 +2,23 @@ import React from 'react';
 import { render, screen } from '@testing-library/react';
 import '@testing-library/jest-dom';
 import Thumbnails from './Thumbnails';
-import { Router, MemoryRouter } from 'react-router';
-import userEvent from '@testing-library/user-event';
+import { MemoryRouter } from 'react-router';
+import { springResident } from '../mockResidentData';
```
Comment (lo-la-do-li) [inline src/Thumbnails/Thumbnails.test.js:6]: Nice import of springResident from mockData!

**Review by cameronRomo — APPROVED** [review]
Very proficient testing here.

File: src/Thumbnails/Thumbnails.test.js [hunk 4]
```diff
@@ -2,19 +2,23 @@ import React from 'react';
...
+          semesterAvailable="Spring-2021"
+        />
+      </MemoryRouter>
     )
-    //execution
-      
-    // assertions
-    expect(screen.getByText()).toBeInTheDocument()
+
+    const doris = screen.getByRole('img', { name: /doris/i })
```
Comment (lo-la-do-li) [inline src/Thumbnails/Thumbnails.test.js:20]: Love Doris

**Review by lo-la-do-li — APPROVED** [review]

File: src/Thumbnails/Thumbnails.test.js (hunk 4 above)
Comment (chadrick-d-dev) [reply src/Thumbnails/Thumbnails.test.js:20]: 🧶🧣👵🏻

File: src/Thumbnails/Thumbnails.test.js (hunk 3 above)
Comment (chadrick-d-dev) [reply src/Thumbnails/Thumbnails.test.js:6]: 💐

**Pull request merged**